src/server_manager/web_app/app.ts

```diff
@@ -673,16 +654,16 @@ export class App {
       userInputConfig = userInputConfig.substr(0, userInputConfig.lastIndexOf('}') + 1);
       serverConfig = JSON.parse(userInputConfig);
     } catch (e) {
-      SentryErrorReporter.logError('Invalid server configuration: could not parse JSON.');
+      console.error('Invalid server configuration: could not parse JSON.');
       return Promise.reject(new Error(''));
     }
     if (!serverConfig.apiUrl) {
       const msg = 'Invalid server configuration: apiUrl is missing.';
-      SentryErrorReporter.logError(msg);
+      console.error(msg);
```

Comment:
Not for this PR, but this seems like a code smell.
It's usually cleaner to only log errors where they are handled, not in multiple steps along the way.

Since we are doing Promise.reject() here, we shouldn't need to call console.error if someone else is handling the errors.

Reply:
Yup. Doing that across our entire codebase will be a big job - hopefully made easier by picking a reasonable Sentry config and sticking with it.